code/game/gamemodes/miniantags/borer/borer_event.dm

```diff
@@ -17,7 +17,10 @@
 		log_and_message_admins("Warning: Could not spawn any mobs for event Borer Infestation")
 
 /datum/event/borer_infestation/start()
-	var/list/vents = get_valid_vent_spawns(exclude_mobs_nearby = TRUE, exclude_visible_by_mobs = TRUE)
+	var/list/vents = get_valid_vent_spawns(exclude_mobs_nearby = TRUE)
+	if(!length(vents))
+		vents = get_valid_vent_spawns(unwelded_only = FALSE, min_network_size = 0)
+		message_admins("Warning: No suitable vents detected for spawning borers. Force picking from station vents regardless of state!")
```

Comment:
nit: i think it's better if the message of "we are about to do a weird thing" happened before the weird thing in question, not after. Mostly because the "weird thing" usually tends to be several lines long, and putting a log message closer to the bottom gets either lost in the code, or not even executed becaues the "weird thing" runtimed or some such.

Shouldn't matter here, of course, I'm just not used to seeing this order.

Reply:
donezo